src/Palindrome.java

```diff
@@ -0,0 +1,21 @@
+import java.util.Scanner;
+
+public class Palindrome {
+    public static boolean isPalindrome(int x) {
+        if (x >= 0) {
```

Comment:
instead of writing else if here you could write `if(x<0) return false;`

Reply:
fixed✅